modules/rollback-operator-upgrades-console.adoc

```diff
@@ -18,73 +19,28 @@ You can roll back the Operator version by using the {ocp} web console.
 
 .Procedure
 
-. Go to the *Operators* -> *Installed Operators* page.
-. Click the {product-title-short} Operator.
-. On the *Operator Details* page, select *Uninstall Operator* from the *Actions* list. Following this action, the Operator stops running and no longer receives updates.
+. In the OpenShift web console, click *Operators* -> *Installed Operators*.
+. Set the *Project* to *rhacs-operator*.
+. Locate the *Advanced Cluster Security for Kubernetes* Operator:
+.. Click the overflow menu {kebab} -> *Uninstall Operator*.
+.. Clear the *Delete all operand instances for this operator* checkbox.
```

Comment:
Can we highlight this step? If the user mistakenly selects the checkbox, it will uninstall ACS completely.